imap_processing/spice/geometry.py

```diff
@@ -27,7 +27,7 @@ class SpiceBody(IntEnum):
     # A subset of IMAP Specific bodies as defined in imap_wkcp.tf
     IMAP = -43
     IMAP_SPACECRAFT = -43000
-    # IMAP Pointing Frame (Despun) as defined in imap_science_0001.tf
+    # IMAP Pointing Frame (Despun) as defined in imap_science_100.tf
```

Comment:
This should be made generic.
```suggestion
    # IMAP Pointing Frame (Despun) as defined in imap_science_###.tf
```